litmus-portal/backend/subscriber/agent/agent.go

```diff
@@ -17,6 +17,7 @@ import (
 var addr = flag.String("addr", "0.0.0.0:8000", "http service address")
 
 func main() {
+	time.Sleep(5 * time.Second)
```

Comment:
Why do we need this??

Reply:
The igc and agent are containers in the same pod, agent is dependant on the igc, hence if igc is not up and running agent will crash. I have added this as a easy way to give igc some time to startup. We will most probably change the subscriber architecture and this will also change in the future.

Reply:
Till that time ... instead of sleeping for 5secs ... Wes should keep a watch over the igc container to check whether it is up or not.
As igc can take more than 5 seconds to be up and running depending on the cluster... So agent will get failed in that case

Reply:
We are removing the agent, there is a PR for that. We don't need this anymore.

Reply:
So will we be removing these changes??

Reply:
@rajdas98 has a PR for that